packages/forms-web-app/src/dynamic-forms/dynamic-components/boolean-text/index.njk

```diff
@@ -1,12 +1,12 @@
 {% extends layoutTemplate %}
 
-{% from "../macros/bullet-list.njk" import bulletList %}
+{% from "macros/bullet-list.njk" import bulletList %}
 {% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
 {% from "govuk/components/radios/macro.njk" import govukRadios %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
 {% from "govuk/components/textarea/macro.njk" import govukTextarea %}
 
-{% set title= question.question + " - Appeal a planning decision - GOV.UK" %}
+{% set title= question.pageTitle + " - Appeal a planning decision - GOV.UK" %}
```

Comment:
Cool, this will help us bing in line the stories and the code base going forwards.